app/src/main/cpp/skyline/input/touch.cpp

```diff
@@ -15,28 +15,34 @@ namespace skyline::input {
         }
     }
 
-    void TouchManager::SetState(const span<TouchScreenPoint> &points) {
+    void TouchManager::SetState(const span<TouchScreenPoint> touchPoints) {
         if (!activated)
             return;
 
         const auto &lastEntry{section.entries[section.header.currentEntry]};
         auto entryIndex{(section.header.currentEntry != constant::HidEntryCount - 1) ? section.header.currentEntry + 1 : 0};
         auto &entry{section.entries[entryIndex]};
+        auto points{touchPoints.subspan(0, std::min(touchPoints.size(), entry.data.size()))};
```

Comment:
I'd say leave the parameter name as `points` and do `points = points.first(std::min(touchPoints.size(), entry.data.size()); // Curb the amount of points to what can fit inside an entry`.

Reply:
Ehh I like how it is, changing parameters is kinda confusing like that

Reply:
`touchPoints` and `points` are not descriptive names or substantially different objects, I feel the semantics of curbing the range make far more sense.

Reply:
I suppose a better variable name then? inputTouchPoints and touchPoints

Reply:
I still prefer the semantics of using `first` as it's already a span. I'd agree if the initial object was a vector but spans already have the semantics of being views into a potentially larger range.

Reply:
first?

Reply:
[`span::first`](https://en.cppreference.com/w/cpp/container/span/first) and assigning to the variable as it's clearly an operation that's curbing the range of a `span` and does directly play into the `span`'s design of being an arbitrary view.

Reply:
Ooo didn't know first existed